app/file.php

```diff
@@ -1,2 +1,50 @@
 <?php
 
+declare(strict_types = 1);
+$root = dirname(__DIR__) . DIRECTORY_SEPARATOR;
+define('FILES_PATH', $root . 'files' . DIRECTORY_SEPARATOR);
+
+class File
+{
+    private string $file_name;
+    private string $file_path;
+    private $raw_data;
+    private array $formatted_data;
+    
+    public function __construct(string $file_name, string $file_path = FILES_PATH)
+    {        
+        $this->file_name = $file_name;        
+        
+        $this->file_path = $file_path;        
+    }   
+
+    public function getFilePath(): string
+    {
+        return $this->file_path;
+    }
+
+    public function getFileName(): string
+    {
+        return $this->file_name;
+    }
+
+    public function toBeExtraced(): string | array 
+    {        
+        if (! file_exists($this->file_path . $this->file_name)){
+               
+            return print('File not found');
```

Comment:
This is good but the idea is to handle failure scenarios with exceptions. If you are not aware how to use or implement them, I can teach you about that.

Reply:
Done